From 7a520f8de53f701652dfa5ae770941fb433a5aca Mon Sep 17 00:00:00 2001
From: Igal Zeltser <izeltser@habana.ai>
Date: Mon, 21 Jun 2021 07:01:54 +0300
Subject: [PATCH] [SW-45099] ppboot: increasing cpu0 stack size

Testing shows that the stack of CPU0 is overruning its allocated size
and corrupting data in the bss section. increasing the stack size to
1024 bytes.

Change-Id: I51d33509bfcc1dda580a2f2f34b1fce35d923d2b
Signed-off-by: Igal Zeltser <izeltser@habana.ai>
---

diff --git a/arch/gaudi/ppboot.lds.S b/arch/gaudi/ppboot.lds.S
index 7d72989..a7e9b77 100644
--- a/arch/gaudi/ppboot.lds.S
+++ b/arch/gaudi/ppboot.lds.S
@@ -30,7 +30,7 @@
 	.stack : {
 		. = ALIGN(8);
 		__stack = . ;
-		. = . + 512 ;
+		. = . + 1024 ;
 		__stack_top = . ;
 	} > .sram
 
